tests/fixtures/application_fixtures.py

```diff
@@ -144,6 +144,15 @@
             integration_config, "fides_example.polling_timeout"
         ),
     },
+    "dynamodb_example": {
```

Comment:
I think these can be removed until we look at shift to use https://hub.docker.com/r/amazon/dynamodb-local/

Reply:
I think it's fine to keep it like this for now, we can come back to it in another ticket